tests/e2e/utils/src/components.js

```diff
@@ -6,7 +6,7 @@
  * Internal dependencies
  */
 import { StoreOwnerFlow } from './flows';
-import modelRegistry from './factories';
+import modelRegistry from '../../factories';
```

Comment:
@tammullen Could this be imported from the package instead of the relative path?

Reply:
It is importing from the `factories.js` within `utils` which itself imports from the factories package. So I think this ok? cc: @ObliviousHarmony as this import was introduced by the e2e factories merge.

Reply:
That sounds okay if @ObliviousHarmony agrees.

Reply:
The `factories.js` file is responsible for setting up and configuring the `@woocommerce/model-factories` package for usage in our E2E test suite. The contents could possibly be put into a utility function and returned instead of exported, if that is preferred, but the package needs to be prepared before consumption either way.